tutorials/io/plot_30_reading_fnirs_data.py

```diff
@@ -74,3 +74,56 @@
              recommended.
 
 """  # noqa:E501
+
+
+
+
+###############################################################################
+# Fake some data
+# --------------
+#
+# Here we just create some fake data with the correct names for an
+# artinis octomon system
+
+import mne
+from mne.channels import make_standard_montage
+from mne.channels.tests.test_standard_montage import _simulate_artinis_octomon
```

Comment:
Obviously this is a big NO. But just wanted to share this small example with Johann as I cant attend sprint in his timezone today.